api/wasm.go

```diff
@@ -318,6 +318,13 @@ type Function interface {
 	//
 	// To safely encode/decode params/results expressed as uint64, users are encouraged to
 	// use api.EncodeXXX or DecodeXXX functions. See the docs on api.ValueType.
+	//
+	// When RuntimeConfig.WithEnsureTermination is toggled, the invocation of this Call method is ensured to terminate
```

Comment:
let's not use different words for close, especially as they don't link to godoc. maybe try to rewrite this so it can link to Close
```suggestion
	// When RuntimeConfig.WithCloseOnContextCancel is toggled, the invocation of this Call method will ... 
```